lib/reform/form/validation/unique_validator.rb

```diff
@@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
     model = form.model_for_property(attribute)
 
     # search for models with attribute equals to form field value
-    query = model.class.where(attribute => value)
+    if options[:case_sensitive] == false
+      downcased_value = value.nil? ? value : value.downcase
+      query = model.class.where("lower(#{attribute}) = ?", downcased_value)
```

Comment:
Fair point... I've just checked the Rails code and it does the comparison like this:

        comparison = if !options[:case_sensitive]
          # will use SQL LOWER function before comparison, unless it detects a case insensitive collation
          klass.connection.case_insensitive_comparison(table, attribute, column, value)
        else
          klass.connection.case_sensitive_comparison(table, attribute, column, value)
        end

File activerecord/lib/active_record/connection_adapters/abstract_adapter.rb, line 259

      def case_insensitive_comparison(table, attribute, column, value)
        table[attribute].lower.eq(table.lower(value))
      end

I can redo the PR using this style of code?